Sources/Brave/BraveSkus/BraveSkusManager.swift

```diff
@@ -155,4 +166,19 @@ public class BraveSkusManager {
       self.expiresAt = expiresAt
     }
   }
+  
+  // MARK: - Session Expired state
+  public static var keepShowingSessionExpiredState = true
```

Comment:
Any reason for making the sessionExpiredProperty Static ?

Is there a way we can add SKUSManager as a dependency in where we used ? Same question applies to method after this also.

Reply:
This is a simple flag to hold one value in-memory without persisting it. BraveSkusManager is a bit heavier class and has entire SKUService setup in the initializer so I would prefer to not pass it around if possible

Why is this flag needed:
If it's on, we show an alert to the user every time they tap on the vpn button. We do it until the app is closed and reopened.
btw this is a bug and the value should be false, left it to true just for testing